main.py

```diff
@@ -0,0 +1,25 @@
+from flask import Flask,render_template,request,redirect
+
+app = Flask(__name__)
+
+@app.route('/')
+def index():
+	return render_template('index.html')
+
+@app.route('/',methods=['POST'])
+def name():
+	user_name = request.form['name']
+	if user_name != "":
+		return render_template('name.html',name=user_name)
+	else:
+		return render_template('error.html')
```

Comment:
it should be `redirect('/errror')'` or `redirect(url_for('error'))`  {you have to import url_for from flask}.
See, if you are returning the template itself `error.html`, its fine(but not a good choice), then you dont need to define error() function below since `@app.route('/error',methods=['POST','GET'])` will never be visited.

Reply:
Ill fix it asap